.circleci/config.yml

```diff
@@ -39,6 +39,12 @@ orbs:
   gcp-cli: circleci/gcp-cli@2.1.0
   revenuecat: revenuecat/sdks-common-config@2.0.0
 
+parameters:
+  action:
+    type: enum
+    enum: [default, bump]
+    default: default
```

Comment:
This comes from https://github.com/RevenueCat/purchases-android/pull/621, because I ran the pipeline in that branch, as soon as that one is merged, we can rebase